### ISSUE TITLE
Compendium of styling and/or functional changes

### ISSUE THREAD

**Issue by toribr** [cityofaustin/techstack#596]
Sketch file: https://drive.google.com/open?id=1S5Ida1oIhDDf9R0RGe_sXbdDllkr8Pju

![content index](https://user-images.githubusercontent.com/3790067/43917967-4e046d42-9bd7-11e8-8fa9-fe59f5759e93.png)

**HEADER & NAV STYLING** (All pages)
Use Sketch file for reference to button colors, text colors, and icons (am working on Style guide to assist)

- [ ] (MVP) Left navigation icons and styling
- [ ] (MVP) Big button styling (primary and secondary)
- [ ] (MVP) Small button styling (index page)
- [ ] (MVP) User name styling
- [ ] (MVP) Base link color #00B6CC
- [ ] (Alpha)Sticky nav - the sticky nav should be sticky only after the user has scrolled away the top header bar (~94 px that contain the seal and user ID).

**Content Index** 
- [ ] (NTH or Alpha) Status link
![Screen Shot 2018-08-09 at 12.36.04 PM.png](https://images.zenhubusercontent.com/5a81e99a4b5806bc2bc6a146/f36271bf-9763-4239-b1ed-5d8ea4fd7180)
I'm not sure why this column would need to link, since there are buttons that link. Let's remove that functionality.
- [ ] (NTH) column headers need new styling {color: #4a4a4a;}
- [ ] (MVP) Can remove "move" from "more" button dropdown
- [ ] (MVP) The "author" column should show the name of the person that authored the page. Currently it's "unknown". (I'm assuming we're not giving everyone a single log-in?)
- [ ] (NTH) Search box needs same height as button
- [ ] (MVP) Fix or don't display search until it works

**Edit Service Page**
![edit page updated](https://user-images.githubusercontent.com/3790067/43971094-78fd28fe-9c95-11e8-8220-5a1c4234b55d.png)
- [ ] (MVP) Header for topic edit should be "Select a topic"
- [ ] (NTH or Alpha) What happened to the SEO tab?
- [ ] (MVP) Header should be "Write any additional content describing the service" with subhead: "Section header: What else do I need to know?" 
![screen shot 2018-08-13 at 1 01 09 pm](https://user-images.githubusercontent.com/3790067/44049168-f9bfb3fc-9ef8-11e8-8967-edcdafb60fdf.png)

- [ ] (MVP) Question mark should align with end of fields
- [ ] (MVP) Steps HTML bar should be restrained to: indent, links, redo, undo. (indent creates a bullet list)
- [ ]  (MVP) Edit page name styling
- [ ] (MVP) Revisions link styling
- [ ] (MVP) While in edit mode, active left nav icon should be content list 
- [ ] (MVP) Button order is Share | Preview | Save Draft v
- [ ] (MVP) Use primary and secondary button styling for edit pages
**SHARE**
- [ ] (NTH) Share should show short URL
- [ ] (NTH) Share notification should be styled as below
![screen shot 2018-08-13 at 10 02 31 am](https://user-images.githubusercontent.com/3790067/44040015-07fb8b12-9ee0-11e8-817c-423913d8fb29.png)
- [ ] (Alpha) Though sharing requires the system to save, it would be preferred that sharing NOT invoke the "draft updated" notification, as it is secondary to the action.  (Also, the button shows saving, which is good enough) 

Preview Panel
- [ ] (MVP) Preview / style guide buttons should show selected state
- [ ] (MVP) Buttons should extend length of panel

**Comment by mateoclarke** [comment]
@toribr You marked this gh-issue as MVP but there are some items in here I know you said you were ok with as-is for MVP. (Ex: sticky head) Can we run through this list together IRL and get clarity about how we prioritize these? must have for mvp, nice to have for mvp, must have for alpha, etc

cc/ @amenity

**Comment by amenity** [comment]
Header & Nav
- [MVP](https://github.com/cityofaustin/techstack/issues/613)
- [Alpha](https://github.com/cityofaustin/techstack/issues/612)
